freqtrade/freqai/data_kitchen.py

```diff
@@ -1027,3 +1002,74 @@ def get_unique_classes_from_labels(self, dataframe: DataFrame) -> None:
         if self.unique_classes:
             for label in self.unique_classes:
                 self.unique_class_list += list(self.unique_classes[label])
+
+# Methods called by interface.py (load_freqai_model())
```

Comment:
I think it might make sense to create a new file for these 2 methods.

that'll keep data_kitchen clean - and focused on the class without containing additional non-class methods.
i know we do this in other cases (exchange) too - but it's not a good pattern - and one i'm trying to get rid of...

Reply:
Done